staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2024,9 +2024,10 @@ type EnvVar struct {
 	// Escaped references will never be expanded, regardless of whether the variable
 	// exists or not.
 	// Defaults to "".
+	// You cannot set both value and valueFrom.
 	// +optional
 	Value string `json:"value,omitempty" protobuf:"bytes,2,opt,name=value"`
```

Comment:
Similar changes needs to be done in corresponding files in staging repo and versioned types.go